app/views/tag_wranglings/_wrangler_dashboard.html.erb

```diff
@@ -1,39 +1,43 @@
-<div id="dashboard" role="navigation region" class="tag wrangling region">
-  <ul class="navigation actions">
-    <% if current_user.is_a?(User) %>
-      <li><%= span_if_current(ts('Wrangling Home'), tag_wrangler_path(current_user)) %></li>
+<% if (logged_in_as_admin? && policy(:wrangling).full_access?) || current_user.try(:is_tag_wrangler?) || @counts %>
+  <div id="dashboard" role="navigation region" class="tag wrangling region">
+    <% if (logged_in_as_admin? && policy(:wrangling).full_access?) || current_user.try(:is_tag_wrangler?) %>
```

Comment:
Based on the comments on this PR and on Jira, the sidebar should be visible to read_access admins:

> I think we can leave the full sidebar visible for anyone with either type of access

Reply:
Might be good to still hide `Wranglers` and `New Tag` from PAC though, since PAC can't access those.